db/crud.go

```diff
@@ -25,7 +25,8 @@ import (
 )
 
 const (
-	kMaxRecentSequences = 20 // Maximum number of sequences stored in RecentSequences before pruning is triggered
+	kMaxRecentSequences            = 20      // Maximum number of sequences stored in RecentSequences before pruning is triggered
+	oversizeSkippedSequenceWarning = 1000000 // Number of skipped sequences to issue a warning for
```

Comment:
I think this number could be a lot smaller.  Any case where a document is allocated a new sequence that's smaller than its previous sequence is unexpected, even at high throughput.  So I think something like 10K would be reasonable.

Reply:
This comment hasn't been addressed yet.